app/services/bgs/dependent_service.rb

```diff
@@ -33,6 +33,7 @@ def get_dependents
     end
 
     def submit_686c_form(claim)
+      Rails.logger.info("BGS::DependentService running! user_uuid=#{uuid} saved_claim_id=#{claim.id} icn=#{icn}")
```

Comment:
Test to double check, but I think this is better to be like

```ruby
Rails.logger.info("BGS::DependentService running!", {user_uuid: uuid, saved_claim_id: claim.id, icn: icn})
```
This will allow it to be a parsed field in the logging tools I believe and comes across in the log a bit cleaner


```ruby
irb(main):009:0> Rails.logger.info("BGS::DependentService running! user_uuid=#{uuid} saved_claim_id=#{claim.id} icn=#{icn}")
2023-05-09 10:36:38.200387 I [40804:59680 (irb):9] Rails -- BGS::DependentService running! user_uuid=ab6b7480-72dc-4e2d-a576-e7a7900d2449 saved_claim_id=565556 icn=100000000                             
irb(main):010:0> Rails.logger.info("BGS::DependentService running!", {user_uuid: uuid, saved_claim_id: claim.id, icn: icn})
2023-05-09 10:36:44.728895 I [40804:59680 (irb):10] Rails -- BGS::DependentService running! -- { :user_uuid => "ab6b7480-72dc-4e2d-a576-e7a7900d2449", :saved_claim_id => "565556", :icn => "100000000" } 
```

Reply:
Same for all the log messages, I think passing in a hash of keys and values is better than putting them in the string